todo_app/app.py

```diff
@@ -30,6 +40,7 @@ def index():  # pylint:disable=unused-variable
     def create_item():  # pylint:disable=unused-variable
         if can_write():
             Items.add_item(request.form['title'])
+            app.logger.info("New item '%s' successfully created.", request.form['title'])
```

Comment:
An example of potentially useful logs here might be in the case that the user _doesn't_ have write permissions - if that's the case then there are reasonable chances that either:
* Our interface is encouraging them to click things they don't have permissions for, which is bad UX
* Or they are trying to get around our security, which is worth being aware of
So adding a warning that this is the case could be a useful thing to look out for